src/Rules/Generators/YieldInGeneratorRule.php

```diff
@@ -65,10 +64,10 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		return [
-			sprintf(
+			RuleErrorBuilder::message(sprintf(
 				'Yield can be used only with these return types: %s.',
 				'Generator, Iterator, Traversable, iterable'
-			),
+			))->line($node->getLine())->build(),
```

Comment:
You can leave RuleErrorBuilder in (since given enough time, I want to all rules return this instance), but remove the line() method call.